compiler-rt/lib/scudo/standalone/allocator_config.def

```diff
@@ -89,6 +89,7 @@ PRIMARY_REQUIRED(const s32, MaxReleaseToOsIntervalMs)
 // Indicates support for offsetting the start of a region by a random number of
 // pages. This is only used if `EnableContiguousRegions` is enabled.
 PRIMARY_OPTIONAL(const bool, EnableRandomOffset, false)
+PRIMARY_OPTIONAL(const s32, DefaultReleaseToOsIntervalMs, 0)
```

Comment:
I think an unset default should be outside the normal range. Maybe something like INT32_MIN. Otherwise you can never default this value to zero.

Reply:
I was thinking to default the value to 5000 (the one used in  release_to_os_interval_ms) when we are ready to deprecate the flag and now the zero means "not set" temporarily. But you're right, especially 0 is useful in many cases.

Reply:
Use INT32_MIN instead.